### PR TITLE
Change My test Farfetch

### PR DESCRIPTION
**Pull request by NatashaRRS**
Мой Первый :) Ура :)

### CODE REVIEW

**Review by AlenaAlexandrovna — APPROVED** [review]

File: src/test/java/school/redrover/FarfetchTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class FarfetchTest {
+
+    @Test
+    public void FarfetchTest () {
+
```
Comment (verafes) [inline src/test/java/school/redrover/FarfetchTest.java:13]: Let's keep just one blank line and remove the rest empty lines. Thanks

File: src/test/java/school/redrover/FarfetchTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://www.farfetch.com/designers/women");
+
+        driver.findElement(By.xpath("//input[@data-testid='search-input']")).sendKeys("Valentino");
+        driver.findElement(By.xpath("//input[@data-testid='search-input']")).click();
+
+//        Thread.sleep (  1000);
```
Comment (verafes) [inline src/test/java/school/redrover/FarfetchTest.java:21]: If we don't need it for testing, let's remove all the commented lines.

**Review by verafes — APPROVED** [review]

**Comment by sweetscripts** [comment]
Congrats! As Sergey mentioned today, I will create a class for our group and ask everyone to add their tests to it.

File: src/test/java/school/redrover/FarfetchTest.java (hunk 1 above)
Comment (NatashaRRS) [reply src/test/java/school/redrover/FarfetchTest.java:13]: Thank you for your recommendation. Appreciate your help.

**Pull request merged**